package.json

```diff
@@ -124,7 +127,8 @@
     "ms-rest-azure": "^2.4.1",
     "request": "^2.81.0",
     "underscore": "1.8.3",
-    "uuid": "^3.0.1"
+    "uuid": "^3.0.1",
+    "vscode": "^1.1.6"
```

Comment:
why do we need vscode as a dependency? Please remove it.

Reply:
done